composer.json

```diff
@@ -58,6 +57,7 @@
         "php-http/cache-plugin": "^1.7",
         "php-http/mock-client": "^1.2",
         "php-http/promise": "^1.0",
+        "phpstan/phpstan-symfony": "^1.2",
```

Comment:
the phpstan-symfony extension is already provided by the github-action docker image.

Reply:
You are right, I did not read the Dockerfile : https://github.com/OskarStark/phpstan-ga/blob/master/Dockerfile#L24.